apps/docs/src/app/core/component-docs/illustrated-message/illustrated-message-docs.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@angular/core';
+
+
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+import * as illustratedMessageSrc from '!raw-loader!./examples/illustrated-message-example.component.html';
+import * as illustratedMessageTsCode from '!raw-loader!./examples/illustrated-message-example.component.ts';
+
+import * as illustratedMessageDialogSrc from '!raw-loader!./examples/illustrated-message-dialog-example.component.html';
+import * as illustratedMessageDialogTsCode from '!raw-loader!./examples/illustrated-message-dialog-example.component.ts';
+
+import * as illustratedMessageSpotSrc from '!raw-loader!./examples/illustrated-message-spot-example.component.html';
+import * as illustratedMessageSpotTsCode from '!raw-loader!./examples/illustrated-message-spot-example.component.ts';
+
+@Component({
+    selector: 'app-illustrated-message',
+    templateUrl: './illustrated-message-docs.component.html'
+})
+export class IllustratedMessageDocsComponent {
+
+    illustratedMessageExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: illustratedMessageSrc,
+            fileName: 'illustrated-message-example'
+        },
+        {
+            language: 'typescript',
+            code: illustratedMessageTsCode,
+            fileName: 'illustrated-message-example'
```

Comment:
please add `component: 'IllustratedMessageExampleComponent'`
the same for other

Reply:
Thanks Yury, I fixed it